x/protocolpool/keeper/keeper.go

```diff
@@ -152,30 +143,29 @@ func (k Keeper) SetToDistribute(ctx context.Context) error {
 		return errorsmod.Wrapf(sdkerrors.ErrUnknownAddress, "module account %s does not exist", types.ProtocolPoolDistrAccount)
 	}
 
-	denom, err := k.stakingKeeper.BondDenom(ctx)
-	if err != nil {
-		return err
-	}
-
 	currentBalance := k.bankKeeper.GetAllBalances(ctx, moduleAccount.GetAddress())
```

Comment:
I guess we get the usual: `WhaT iF tHe modUle aCCounT Has ManY DeNOM?`
Which is that case will be relatively valid. Any way we can make the claimer pay for this call?
As this is call in begin block maybe not, but what prevents in a token factory enabled chain, to send to the module accounts 1 millions worthless tokens until it slows the chain.

Unless we make `ProtocolPoolDistrAccount` a blocked module account, like FeeCollector is.

Reply:
> As this is call in begin block maybe not, but what prevents in a token factory enabled chain, to send to the module accounts 1 millions worthless tokens until it slows the chain.

Yeah, I thought the same, realistically it shouldn't be needed but it could make things go wrong in the long run, so I guess I'll add the whitelist now :/